Sources/TuistKit/Models/BuildConfiguration.swift

```diff
@@ -13,3 +13,19 @@ extension BuildConfiguration: XcodeRepresentable {
         }
     }
 }
+
+struct ConfigurationList: Sequence {
```

Comment:
Not sure this is required, the original implementation added some extra meta data here. Shall we remove it and just use an array `[Configuration]`

Reply:
What was the metadata added @ollieatkinson?

Reply:
the root project info